tests/console/yast2_apparmor.pm

```diff
@@ -0,0 +1,143 @@
+    wait_still_screen;
+    send_key 'alt-t';
+    assert_screen 'yast2_apparmor_profile_mode_configuration_toggle';
+    wait_still_screen;
+    send_key 'alt-b';
+    wait_still_screen;
+
+    # close apparmor configuration
+    send_key 'alt-d';
+    wait_still_screen;
+    assert_script_run("systemctl show -p ActiveState apparmor.service | grep ActiveState=active");
+
+    # part 2: start apparmor configuration again
+    script_run("/sbin/yast2 apparmor; echo yast2-apparmor-status-\$? > /dev/$serialdev", 0);
+    assert_screen 'yast2_apparmor';
+    send_key 'down';
+    assert_screen 'yast2_apparmor_configuration_manage_existing_profiles';
+    send_key 'ret';
+    assert_screen 'yast2_apparmor_configuration_manage_existing_profiles_edit';
+    send_key 'alt-p';
+    send_key 'down';
+
+    # add a profile
+    assert_screen 'yast2_apparmor_configuration_manage_existing_profiles_edit_add';
+    send_key 'alt-i';
+    wait_still_screen;
+    send_key 'alt-a';
+    wait_still_screen;
+    send_key 'alt-f';
+    assert_screen 'yast2_apparmor_configuration_manage_existing_profiles_edit_add_file';
+    send_key 'ret';
+    send_key 'alt-e';
+    type_string 'I_added_this_profile';
+    send_key 'alt-p';
+    wait_still_screen;
+    send_key 'down';
+    send_key 'up';
+    send_key 'spc';
+    send_key 'down';
+    send_key 'spc';
+
+    # confirm with cancel
+    send_key 'alt-c';
+    wait_still_screen;
+
+    # now add a directory with permission for read and write
+    send_key 'alt-a';
+    wait_still_screen;
+    send_key 'alt-d';
+    send_key 'alt-e';
+    type_string '/tmp';
+    sleep 2;
+    send_key 'alt-p';
+    wait_still_screen;
+    send_key 'down';
+    send_key 'up';
+    send_key 'spc';
+    send_key 'down';
+    send_key 'spc';
+    send_key 'alt-o';
+    wait_still_screen;
+    send_key 'alt-d';
+
+    # confirm to save changes to the profile
+    for (assert_screen 'yast2_apparmor_configuration_manage_existing_profiles_edit_file_changed') { send_key 'alt-y'; }
+
+    # close now AppArmor configuration
+    sleep 3;
+    send_key 'alt-n';
+    sleep 2;
+
+    # part 3: manually add profile
+
+    # prepare a new profile at first
+    script_run("cp /etc/apparmor.d/sbin.syslogd /new_profile");
+
+    #start apparmor configuration again
+    script_run("/sbin/yast2 apparmor; echo yast2-apparmor-status-\$? > /dev/$serialdev", 0);
+    assert_screen 'yast2_apparmor';
+    send_key 'down';
+    send_key 'down';
+    send_key 'ret';
+    assert_screen 'yast2_apparmor_configuration_add_profile';
+    send_key 'alt-f';
+    for (1 .. 30) { send_key 'backspace'; }
+    type_string '/new_profile';
+    send_key 'alt-o';
+    wait_still_screen;
+
+    # Add entry for new profile
+    send_key 'alt-a';
+    send_key 'alt-r';
+    send_key 'alt-s';
+    send_key 'alt-d';
+    # confirm to save changes to profile
+    for (assert_screen 'yast2_apparmor_configuration_manage_existing_profiles_edit_file_changed') { send_key 'alt-y'; }
```

Comment:
@okurz okay, will change sleep into wait_still_screen and define seconds for it as well.